src/object/srv_mod.c

```diff
@@ -153,7 +153,7 @@ struct dss_module_key obj_module_key = {
 };
 
 static struct dss_module_ops ds_obj_mod_ops = {
-	.dms_abt_pool_choose_cb = ds_obj_abt_pool_choose_cb,
+	.dms_abt_pool_choose_cb = NULL,
```

Comment:
if choose_cb is not being used, then this callback can be totally removed.

Reply:
I kept it because I'm not sure if it could be used when we improve the scheduler in the future.